src/smart-components/Thread/components/ThreadList/index.tsx

```diff
@@ -57,6 +58,8 @@ export default function ThreadList({
   return (
     <div className={`sendbird-thread-list ${className}`}>
       {allThreadMessages.map((message, idx) => {
+        // @ts-ignore
```

Comment:
It seems the similar code in other files don't have this. e.g. https://github.com/sendbird/sendbird-uikit-react/pull/476/files#diff-f6ecc596d3e1dd94913459ff57d72848c6185c6c18acf14dc737b364926e9998R73 
What does it say if we don't have this?

Reply:
ah is my mistake(will remove)